src/Model/Actions.re

```diff
@@ -55,6 +56,9 @@ type t =
   | TextInput([@opaque] Revery.Events.textInputEvent)
   | HoverShow
   | ChangeMode([@opaque] Vim.Mode.t)
+  | ContextMenuUpdate([@opaque] ContextMenu.t(t))
```

Comment:
`ContextMenuUpdate` doesn't sound like an event to me (it sounds like a description of what _will_ happen) - it sounds like the 'function call` paradigm. Would `ContextMenuUpdated` make more sense?

Reply:
Yep, definitely should be. Good catch!

Reply:
Cool! Starting to get the hang of it, hopefully 😅